support/src/pages/base-de-connaissance/[slug].js

```diff
@@ -12,7 +13,9 @@ const Content = () => {
     setSlug(router.query?.slug || "");
   }, [router.query?.slug]);
 
-  const { data: response } = useSWR(API.getUrl({ path: `/support-center/knowledge-base/${slug}` }));
+  const { user } = useUser(); // find the user in some way, with the cookie ?
```

Comment:
Le commentaire est obsolète non ?

```suggestion
  const { user } = useUser();
```